src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/configure-settings.mdx

```diff
@@ -978,12 +979,13 @@ If you modify any of settings below, be sure to call the feature flag before the
             <td>
                 Enable or disable offline data storage when no internet connection is available. To configure the amount of offline storage, see [Set max offline storage size](/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/set-max-offline-storage/#ios).
 
-                Default value is `true`.
-                
             </td>
             <td>
+
+                Offline storage is disabled by default. To enable it, add the following feature flag:
+
                 ```swift
-                NewRelic.disableFeatures(NRMAFeatureFlags.NRFeatureFlag_OfflineStorage)
+                NewRelic.enableFeatures(NRMAFeatureFlags.NRFeatureFlag_OfflineStorage)
```

Comment:
@ndesai-newrelic Same question here

Reply:
just this is required NewRelic.enableFeature(FeatureFlag.OfflineStorage)